aca_weekly_report/report.py

```diff
@@ -536,6 +540,8 @@ def make_metric_print(dat, warn_map):
 
     print_table = {}
     # Add some long URL fields to the dictionary before making an astropy.table
+    print_table["load"] = (
+        f"<A HREF='https://icxc.cfa.harvard.edu/mp/mplogs{dat['mp_dir']}starcheck.html'>{dat['load_name']}</A>")
```

Comment:
Why not link directly to the obsid with an additional `#obsid{obsid}` (or something like that)?

Reply:
Sure.  I was more interested the load than the catalog but no problem to do the most useful thing here with the link.